src/ZoweExplorerExtender.ts

```diff
@@ -70,9 +90,15 @@ export class ZoweExplorerExtender implements ZoweExplorerApi.IApiExplorerExtende
      * might want to request that profiles should get reloaded
      * to make them automatically appears in the Explorer drop-
      * down dialogs.
+     *
+     * @param profileType optional profile type that the extender can specify
      */
-    public async reloadProfiles(): Promise<void> {
+    public async reloadProfiles(profileType?: string): Promise<void> {
         // sequentially reload the internal profiles cache to satisfy all the newly added profile types
-        ZoweExplorerExtender.refreshProfilesQueue.add( () => Profiles.getInstance().refresh());
+        await ZoweExplorerExtender.refreshProfilesQueue.add( () => Profiles.getInstance().refresh());
+        // profileType is used to load a default extender profile if no other profiles are populating the trees
+        this.datasetProvider?.addSession(undefined, profileType);
+        this.ussFileProvider?.addSession(undefined, profileType);
+        this.jobsProvider?.addSession(undefined, profileType);
```

Comment:
For future reference, let's try to avoid doing this.
I'm sure you don't want to be in a situation where you have to do `addSession(undefined, type, null, false, null, true)` :yum:

Reply:
Currently, I don't foresee more parameters getting passed to `addSession()`, so I think we can leave this as-is for now. However, going forward, if more optional parameters are added to `addSession()`, then we can definitely look at passing them in a different way (e.g. an interface). Thank you for this feedback, @zFernand0!